src/app/config/projectsJsonData.ts

```diff
@@ -100,13 +100,16 @@ export default class projectsJsonData {
     return undefined;
   }
 
-  getProjectRepoAndBranch(projectTypeKey: string, scriptType: string, prerelease: boolean) {
+  getProjectRepoAndBranch(projectTypeKey: string, scriptType: string, prerelease: boolean, isJsonProject: boolean) {
```

Comment:
If you use the right branch URL in the ProjectProperties.json then you shouldn't have to use a different boolean here to override the branch name.

Reply:
Then I would have a condition for this function to not get called. As this functions returns something that will get appended to the URL. So we cannot just put the URL as it is without some scape path for the JSON condition

Reply:
This is returning information about the template repo and branches.  I don't see where it's getting appended, but it is getting it's properties looked up.

Reply:
This is where the branch gets appended and the URL gets changed:
```
    export async function downloadProjectTemplateZipFile(projectFolder: string, projectRepo: string, projectBranch: string): Promise<void> {
        const projectTemplateZipFile = `${projectRepo}/archive/${projectBranch}.zip`;
```
This is in `src\app\helpers\helperMethods.ts`. As we can see, the URL `projectRepo` gets changed and it not what is actually used.

Reply:
I see.  So yes, the base repo should be the base url w/out the branch . . . but the branches should point to yo-office and yo-ffice-prerlease branches for the new template.

Reply:
I just changed the branch names in the `.json` config file to the new `json` branch.
How can I make the branches point to `yo-office` as it is not a new repository? We need to make it link to the associated json branch

Reply:
Looks at the CF repo for the example (shared app branches).  The core template is the root + yo-office + yo-office-prerelease branches, and the new template uses the json + json-yo-office + json-yo-office-prerelease branches.  Generator office doesn't reference the root branch directly, just the other two branches designated in the properties json file.  This is how we control what is in use vs still in "development".  Updating these is part of the repo merge process I go through.

Reply:
I just created those 2 braches and updates the code to direct to them